components/feature/addons/src/main/res/layout/mozac_feature_addons_item.xml

```diff
@@ -16,6 +16,8 @@
 
     <androidx.appcompat.widget.AppCompatImageView
         android:id="@+id/add_on_icon"
+        android:layout_width="@dimen/mozac_widget_favicon_size"
+        android:layout_height="@dimen/mozac_widget_favicon_size"
         style="@style/Mozac.Widgets.Favicon"
```

Comment:
lint issue? That's unfortunate as this way we will have to update `width` and `height`, when these values change on `Mozac.Widgets.Favicon`

Reply:
Yeah, lint doesn't seem to recognize width/height from the style. I'll check if there's an option to just suppress.

Reply:
I removed it again and replaced it with `tools:ignore="RequiredSize"`